frozen-abi/src/abi_digester.rs

```diff
@@ -36,8 +36,8 @@ pub enum DigestError {
 }
 
 impl SerdeError for DigestError {
-    fn custom<T: std::fmt::Display>(_msg: T) -> DigestError {
-        unreachable!("This error should never be used");
+    fn custom<T: std::fmt::Display>(msg: T) -> DigestError {
+        panic!("Unexpected SerdeError: {msg}");
```

Comment:
haha, certainly not related to the syn update. but i like your careful eye and some love. :)

Reply:
Ah, this got it by accident, actually.
When I was debugging one test failure, I was getting this panic triggered.
Not seeing the original message was making it harder.
It happens if the macro does not work correctly, actually including skipped fields in the hash calculation.